docs/install.rst

```diff
@@ -163,7 +163,18 @@ Then, delete all data associated with your Open edX platform::
 Finally, uninstall Tutor itself::
 
     # If you installed tutor from source
-    pip uninstall tutor
+    pip uninstall -y tutor
+    pip uninstall -y tutor-android
```

Comment:
How about?
```
pip uninstall -y <plugin_name>
```

Or are those specific plugins installed?

Reply:
Those specified in https://github.com/overhangio/tutor/blob/master/setup.py#L62 and https://github.com/overhangio/tutor/blob/master/requirements/plugins.txt will be installed when installing using the recommended approach `pip install tutor[full]`

Reply:
I'd rather not give the full list of plugins here, it's just too cumbersome. Moreover, it's not _strictly_ necessary to uninstall plugins to "get rid" of tutor. Thus, I suggest that we give generic instructions on how to uninstall plugins. Something along those lines:

> List all Tutor plugins
> 
>     $ pip freeze | grep tutor
> 
> Then, uninstall every plugin:
> 
>     $ pip uninstall <tutor plugin name>